tests/deb/deb_tests.bzl

```diff
@@ -15,6 +15,25 @@
 
 load("@bazel_skylib//lib:unittest.bzl", "analysistest", "asserts")
 
+def assert_contains(env, expected, got, msg = None):
+    """Asserts that the given `expected` occurs in a set of things.
```

Comment:
Nit: "set" could refer to skylib [sets](https://github.com/bazelbuild/bazel-skylib/blob/main/docs/new_sets_doc.md), which are `struct`s.

Maybe say:

```markdown
Asserts that the given `expected` occurs in iterable (`list`, `dict`) `got`.
```

Reply:
done